content/modules/ROOT/pages/self-service-patterns/lab-namespace-self-service-patterns.adoc

```diff
@@ -1,175 +1,159 @@
 
 == Introduction
 
-Welcome to this lab, where you will have the chance to learn how to work with Software templates in Red Hat Developer Hub, and create a new OpenShift Project from Red Hat Developer Hub with custom configurations.
+In this exercise you will learn how to work with Software templates in Red Hat Developer Hub and create a new OpenShift Project (also referred to as a _namespace_) from Red Hat Developer Hub with custom configurations.
+
+This lab will use an existing template to create OpenShift Projects. You will work on increasing functionality by updating the software template to limit resource quotas.
 
-This lab will use an existent template to create OpenShift Projects. You will be working to increasing functionality by updating the software template to limit resource quotas.
-During this lab, you will have the opportunity to explore different projects' customizations across different clusters.
+During this lab, you will have the opportunity to explore different project customizations across different clusters.
 
 
 [#lab]
 == Scenario
-Now, imagine for a moment that you need to fullfil a need for the Python Development team. The Python Team's responsibility is to create a new start prototyping with AI and for this you need a namespace, this new namespace is not part of the Software Development Lifecycle. The request is having a namespace isolated from the rest of the cluster, with resource limits but enough resources to experiment.
+Now, imagine for a moment that you need to fulfill a need for the Python Development team. The Python Team needs the ability to create new namespaces to start prototyping with AI, and these new namespaces are not part of the Software Development Lifecycle. These new namespaces should be isolated from the rest of the cluster, with resource limits but sufficient resources for experimentation.
 
 
 == Explore the Project template in GitLab
-. Take the time to review the source code to get familiarize with the content.
-
-
-* {gitlab_url}/rhdh/rhads-enablement-l3-st-self-service/-/blob/main/namespace-medium[{gitlab_url}/rhdh/rhads-enablement-l3-st-self-service/-/blob/main/namespace-medium,window=_blank]
-
-** Login with your user credentials:
+. Take the time to {gitlab_url}/rhdh/rhads-enablement-l3-st-self-service/-/blob/main/namespace-medium[review existing template^] in GitLab to get familiar with the content. If you need to log in, use your GitLab credentials:
 
     ** *Username*: {gitlab_user}
     ** *Password*: {gitlab_user_password}
 
+Open the following files or folders to examine their contents:
 
-* Component/catalog-info.yaml: This file will represent the object in Developer Hub, such as name and links.
-+
-image:self-service-patterns/namespace-lab/source-code-catalog-info.png[Project Catalog]
-
-* Developer Hub, powered by GitOps, maintains the desired state as the actual state in the cluster for any template definitions, ensuring system stability.
```

Comment:
add the information about GitOps that was removed.
* Developer Hub, powered by GitOps, maintains the desired state as the actual state in the cluster for any template definitions, ensuring system stability